diagrams/zkp-cred-pres.svg

```diff
@@ -1 +1,136 @@
534c0 10.064819 -8.159119 18.223999 -18.223938 18.223999l-251.6781 0l0 0c-10.064819 0 -18.223938 -8.15918 -18.223938 -18.223999z" fill-rule="evenodd"/><path fill="#000000" d="m740.1947 648.10297l4.9375 0q1.65625 0 2.53125 0.265625q1.1875 0.34375 2.015625 1.234375q0.84375 0.875 1.28125 2.15625q0.4375 1.28125 0.4375 3.15625q0 1.640625 -0.40625 2.84375q-0.5 1.453125 -1.4375 2.359375q-0.703125 0.671875 -1.890625 1.0625q-0.890625 0.28125 -2.390625 0.28125l-5.078125 0l0 -13.359375zm2.703125 2.265625l0 8.84375l2.015625 0q1.125 0 1.625 -0.125q0.65625 -0.171875 1.078125 -0.5625q0.4375 -0.390625 0.703125 -1.28125q0.28125 -0.90625 0.28125 -2.453125q0 -1.546875 -0.28125 -2.375q-0.265625 -0.828125 -0.765625 -1.28125q-0.484375 -0.46875 -1.234375 -0.640625q-0.5625 -0.125 -2.21875 -0.125l-1.203125 0zm16.363586 8.015625l2.546875 0.421875q-0.484375 1.40625 -1.546875 2.140625q-1.0625 0.734375 -2.65625 0.734375q-2.515625 0 -3.734375 -1.65625q-0.953125 -1.3125 -0.953125 -3.328125q0 -2.40625 1.25 -3.765625q1.265625 -1.359375 3.1875 -1.359375q2.15625 0 3.40625 1.421875q1.25 1.421875 1.1875 4.375l-6.40625 0q0.03125 1.140625 0.609375 1.78125q0.59375 0.625 1.484375 0.625q0.59375 0 1.0 -0.328125q0.421875 -0.328125 0.625 -1.0625zm0.15625 -2.59375q-0.03125 -1.109375 -0.578125 -1.6875q-0.546875 -0.578125 -1.328125 -0.578125q-0.84375 0 -1.390625 0.609375q-0.546875 0.609375 -0.53125 1.65625l3.828125 0zm7.0787964 5.671875l-2.5625 0l0 -9.671875l2.375 0l0 1.375q0.609375 -0.984375 1.09375 -1.28125q0.484375 -0.3125 1.109375 -0.3125q0.875 0 1.6875 0.484375l-0.796875 2.234375q-0.640625 -0.421875 -1.203125 -0.421875q-0.53125 0 -0.90625 0.296875q-0.375 0.296875 -0.59375 1.078125q-0.203125 0.765625 -0.203125 3.234375l0 2.984375zm4.80719 -10.984375l0 -2.375l2.5625 0l0 2.375l-2.5625 0zm0 10.984375l0 -9.671875l2.5625 0l0 9.671875l-2.5625 0zm7.8395386 0l-3.90625 -9.671875l2.6875 0l1.828125 4.9375l0.53125 1.640625q0.203125 -0.625 0.265625 -0.828125q0.125 -0.40625 0.265625 -0.8125l1.84375 -4.9375l2.625 0l-3.84375 9.671875l-2.296875 0zm13.313232 -3.078125l2.546875 0.421875q-0.484375 1.40625 -1.546875 2.140625q-1.0625 0.734375 -2.65625 0.734375q-2.515625 0 -3.734375 -1.65625q-0.953125 -1.3125 -0.953125 -3.328125q0 -2.40625 1.25 -3.765625q1.265625 -1.359375 3.1875 -1.359375q2.15625 0 3.40625 1.421875q1.25 1.421875 1.1875 4.375l-6.40625 0q0.03125 1.140625 0.609375 1.78125q0.59375 0.625 1.484375 0.625q0.59375 0 1.0 -0.328125q0.421875 -0.328125 0.625 -1.0625zm0.15625 -2.59375q-0.03125 -1.109375 -0.578125 -1.6875q-0.546875 -0.578125 -1.328125 -0.578125q-0.84375 0 -1.390625 0.609375q-0.546875 0.609375 -0.53125 1.65625l3.828125 0zm13.500671 5.671875l-2.390625 0l0 -1.421875q-0.59375 0.828125 -1.40625 1.234375q-0.796875 0.40625 -1.609375 0.40625q-1.671875 0 -2.859375 -1.34375q-1.1875 -1.34375 -1.1875 -3.75q0 -2.453125 1.15625 -3.734375q1.15625 -1.28125 2.921875 -1.28125q1.625 0 2.8125 1.34375l0 -4.8125l2.5625 0l0 13.359375zm-6.84375 -5.046875q0 1.546875 0.4375 2.234375q0.609375 1.015625 1.71875 1.015625q
```

Comment:
The intention for this bit of the example was not for it to be a predicate, but rather to show that the property names in the presented derived credential may not match the property names in the original credential.
so rather than "Has a Degree" I would prefer it be kept as "degree"

Reply:
something strange is happening. I'm trying to comment on line 115, but  my comment is being assigned to line 1 and immediately showing as outdated once I finish my review.

Reply:
somehow must be registering your selection of line in the wrong spot.

Anyway, I see the comments, and I am happy to make the change.

Reply:
done in 3b6f73c